packages/hoprd/package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "clean": "rimraf ./lib ./tsconfig.tsbuildinfo",
     "build": "yarn clean && tsc -p . && yarn buildAdmin && chmod u+x lib/*.js",
-    "buildAdmin": "next build hopr-admin",
+    "buildAdmin": "HOPRD_GIT_COMMIT=$(git rev-parse --short HEAD) next build hopr-admin",
```

Comment:
Don't really like to have sub-shells in npm scripts, but also don't mind too much either.